frontend/src/app/specification-form/specification-form.component.html

```diff
@@ -13,10 +13,10 @@ <h4 class="modal-title">Local file upload</h4>
        <div class="alert alert-danger" role="alert" *ngIf="error">
          {{ error }}
        </div>
-       <input type="file" placeholder="Upload Swagger Specification" accept=".yml,.json,.yaml,.graphql" ngModel
+       <input multiple type="file" accept=".yml,.json,.yaml,.graphql" ngModel
```

Comment:
I think I'd still like a placeholder, if possible

Reply:
This placeholder never actually appeared 😄 The placeholder attribute is only used for text input types